apis/integreatly/v1alpha1/types/types.go

```diff
@@ -27,6 +27,8 @@ type ResourceTypeSpec struct {
 	Tier       string     `json:"tier"`
 	SkipCreate bool       `json:"skipCreate,omitempty"`
 	SecretRef  *SecretRef `json:"secretRef"`
+	RemoveBlockOnPublicAccess  bool `json:"removeBlockOnPublicAccess,omitempty"`
+	AllowPublicGetObjectAccess  bool `json:"allowPublicGetObjectAccess,omitempty"`
```

Comment:
normally we use the strategy config pass configuration to resources in cro, this pattern goes against how we handle all other resources. I don't know the longevity of cro, considering things like crossplane. I dont have a strong opinion, only If cro is staying around for the long term I would think it is best use a strat config to get this configuration. 

Maybe @pmccarthy  can give some indication here, is it worth updating to match other resources, or is it okay as is? I am conscious of time to refactor.

Reply:
The reason for this approach was to modify one specific blobStorage/S3 bucket as apposed to all via the strategy map. However, we recently added the ability to overwrite aws strategies on a per instance basis for Postgres and Redis to integreatly. We could add code to do the same for BlobStorage and then take the approach suggested. 
@pmccarthy Do you have a preference on approach?

Reply:
@briangallagher @ciaranRoche  I don't have a particularly strong preference either. If there is an immediate need to have the permissions updated for specific customers then we should try make CRO flexible enough to support per instance configs, based on the work Brian has done recently. However, if this change is something we plan to roll out to all customers across the fleet then there's no need to add per instance config support to S3 IMO.